exercises/identify-conflicts.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Identify Conflicts
+tags:
+  - exercise
+  - converge
+---
+Throughout Diverge there should have been many conflicting ideas. A conflict is a place where there are two or more different approaches to solving the same problem. Conflicting approaches are helpful because they illuminate possible choices for your product.
```

Comment:
```suggestion
Throughout Diverge there should have been many conflicting ideas. A conflict is a place where there are two or more different approaches to solving the same problem. Conflicting approaches are helpful because they illuminate possible choices for your product.

## Requirements

- **Estimated time needed**: 20 min
- **Team**: Facilitator, Note-taker, Stakeholders
 
 ## Instructions
 
1. Review the [Story Boards](/exercises/3-step-storyboards) and identify where there are multiple approaches to solving the same problem.
2. Talk through the different approaches start determining which it the best approach as you move into creating the [Final Storyboard](/exercises/make-a-map.md)
```